sonarts-core/src/cfg/cfg.ts

```diff
@@ -69,6 +69,16 @@ export class ControlFlowGraph {
             );
           }
         }
+        if (block.branchingElement) {
```

Comment:
looks smelly

Reply:
@vilchik-elena this was due to the branchingElement being associated to the top of a condition expression (which can be a non-branching block) instead of the bottom, so an empty non-branching block could contain a branchingElement. Fixed it by making sure the branchingElement is always associated to the branching block (at the constructor), have a look at my next commit.